include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -4670,25 +4670,29 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
                    VectorizedArrayType::size()>
           vector_ptrs = {};
 
-        (void)comp;
-
-        /*
         for (unsigned int v = 0; v < n_filled_lanes; ++v)
           {
+            Assert(ind < this->dof_info->dof_indices_contiguous_sm.size(),
+                   ExcNotImplemented());
+            Assert(this->cell * VectorizedArrayType::size() + v <
+                     this->dof_info->dof_indices_contiguous_sm[ind].size(),
+                   ExcNotImplemented());
```

Comment:
Are these really not-implemented cases or rather out of range, i.e. `AssertIndexRange()`?

Reply:
Don't blame me. Blame `github-linux / linux debug cuda-10`:

```
/home/runner/work/dealii/dealii/include/deal.II/matrix_free/fe_evaluation.h: In lambda function:
/home/runner/work/dealii/dealii/include/deal.II/matrix_free/fe_evaluation.h:4675:187: error: base operand of ‘->’ is not a pointer
             AssertIndexRange(ind,
                                                                                                                                                                                           ^ 
/home/runner/work/dealii/dealii/include/deal.II/matrix_free/fe_evaluation.h:4675:187: error: base operand of ‘->’ is not a pointer
/home/runner/work/dealii/dealii/include/deal.II/matrix_free/fe_evaluation.h:4675:236: error: template argument 2 is invalid
             AssertIndexRange(ind,
```

See the second commit^^

Reply:
Does it work if you put the arguments into additional braces?

Reply:
Let's see.

Reply:
Any other ideas?

Reply:
How about writing out the check, i.e.,
```
   Assert(ind < this->dof_info->dof_indices_contiguous_sm.size(),
              ExcIndexRange(ind, 0, this->dof_info->dof_indices_contiguous_sm.size()));
```
In any case, an annoying error.

Reply:
Next round...

Reply:
OK. Then let me squash the commits.